OMCompiler/Compiler/Template/CodegenFMUCommon.tpl

```diff
@@ -58,10 +58,11 @@ template ModelExchange(SimCode simCode, list<String> sourceFiles)
 match simCode
 case SIMCODE(__) then
   let modelIdentifier = modelNamePrefix(simCode)
-  let pdd = if providesDirectionalDerivative(simCode) then ' providesDirectionalDerivative="true"' else ''
+  let pdd = if providesDirectionalDerivative(simCode) then 'providesDirectionalDerivative="true"' else 'providesDirectionalDerivative="false"'
```

Comment:
Please don't print the default values. But you can use a flag (`-d=force-fmi-attributes`) if you want to force all the attributes including default values, e.g., for debugging purpose.

Reply:
Okay